src/components/SignIn/index.jsx

```diff
@@ -81,15 +81,19 @@ const SignIn = () => {
   return (
     <div className="container">
       <h4 className="welcomeBack">{t("signIn.welcomeBack")}</h4>
+      <p>Sign in with</p>
       <div>
-        <Button onClick={handleGoogleLogin} variant="outline-primary">
-          <FontAwesomeIcon icon={faGoogle} />
-        </Button>
-        <Button onClick={handleFacebookLogin} variant="outline-primary">
-          <FontAwesomeIcon icon={faFacebook} />
-        </Button>
+        <div className="icons">
+          <Button className="socialIcons" onClick={handleGoogleLogin} variant="danger">
+            <FontAwesomeIcon icon={faGoogle} />
+          </Button>
+          <Button className="socialIcons" onClick={handleFacebookLogin}>
+            <FontAwesomeIcon icon={faFacebook} />
+          </Button>
+        </div>
       </div>
       <Form id="signInForm" onSubmit={handleSubmit}>
+        <p>or</p>
         <Form.Group as={Row} controlId="formBasicEmail">
           <Form.Label></Form.Label>
```

Comment:
I think you forgot to write the email in the label

Reply:
> I think you forgot to write the email in the label

It must already been fixed, i don't know why is it here